pyproject.toml

```diff
@@ -1,3 +1,37 @@
 [tool.black]
 line-length = 88
 target-version = ['py37', 'py38']
+
+[tool.poetry]
+name = "forenings_medlemmer"
```

Comment:
I stedet for requriements.txt har vi nu den her oversigt. 
Alle pakker er opdateret undtagen quickpay